ui/desktop/app/templates/scopes/scope/projects/targets/index.hbs

```diff
@@ -108,10 +108,11 @@
               <B.Td>
                 {{#if B.data.sessions.length}}
                   <Hds::Button
+                    data-test-targets-sessions-flyout-button={{B.data.target.id}}
                     @text={{t 'actions.yes'}}
                     @color='tertiary'
                     @icon='info'
-                    {{on 'click' this.toggleSessionsFlyout}}
+                    {{on 'click' (fn this.selectTargetSessions B.data)}}
```

Comment:
Why did we pass all the targets instead of just the target we needed here?

Reply:
`B.data` is an object that includes a single target and associated sessions.